molo/commenting/views.py

```diff
@@ -130,3 +130,24 @@ def get(self, request, parent_comment_pk):
             'form': form,
             'comment': comment,
         })
+
+
+class WagtailCommentReplyView(FormView):
```

Comment:
Please rename to `MoloCommentsAdminReplyView`, rename the associated template and move to `admin_views.py`